packages/melody-idom/src/dom_util.ts

```diff
@@ -69,6 +69,28 @@ const getActiveElement = function(node) {
     return isDocumentRoot(root) ? root.activeElement : null;
 };
 
+/**
+ * Function that checks whether some element is contained inside a node. This function
+ * has a fallback implementation for cases where Node.prototype.contains is
+ * not implemented (e.g. IE11 for SVGElements).
+ * @param {!Node} node The node that should be parent of elm.
+ * @param {?Element} elm The element that should be child of elm.
+ * @return {Boolean} whether or not elm is contained within node.
+ */
+const nodeContainsElm = function(node, elm) {
```

Comment:
How about `isInElement`? At the very least we should change `Elm` for `Element`. The minifier will take care of mangling the name for us.